demo/admin/src/redirects/RedirectsDependency.tsx

```diff
@@ -0,0 +1,16 @@
+import { createDependencyMethods, type DependencyInterface } from "@comet/cms-admin";
+import { LinkBlock } from "@src/common/blocks/LinkBlock";
+import { FormattedMessage } from "react-intl";
+
+export const RedirectDependency: DependencyInterface = {
+    displayName: <FormattedMessage id="redirect.displayName" defaultMessage="Redirect" />,
+    ...createDependencyMethods({
+        rootQueryName: "redirect",
+        rootBlocks: {
+            target: {
+                block: LinkBlock,
```

Comment:
this LinkBlock is not the one used by redirects, currently that one is created inside createRedirectsPage and you don't have access to it. This somehow needs to be refactored.

Reply:
imo it would be okay for now to use it here and add a todo. We probably need to refactor the redirects module to define the block in the application (api and admin), which would be a breaking change.

Reply:
Alternatively, we can export a function to create the redirects link block, create it in the application, but only use it here. Later we can then refactor the redirects module to also use it.

Reply:
changed in: 359268021d8415c468305a04497c34b291188236

Did I understand that correctly or did you mean it another way?